scripts/webpack/webpack.common.js

```diff
@@ -17,6 +16,26 @@ class CopyUniconsPlugin {
   }
 }
 
+// JOSH: use copy-webpack-plugin?
+class CopyMonacoPlugin {
+  apply(compiler) {
+    function copy() {
+      let srcDir = path.resolve(__dirname, '../../node_modules/monaco-editor/dev/vs');
+      let destDir = path.resolve(__dirname, '../../public/build/monaco/min/vs');
+      console.log('**** copying from', srcDir, 'to', destDir);
+      fs.copySync(srcDir, destDir);
+
+      let srcDir2 = path.resolve(__dirname, '../../node_modules/@kusto/monaco-kusto/release/min/');
+      let destDir2 = path.resolve(__dirname, '../../public/build/monaco/min/vs/language/kusto');
+      console.log('**** copying from', srcDir2, 'to', destDir2);
+      fs.copySync(srcDir2, destDir2);
+    }
+
+    compiler.hooks.run.tap('CopyMonacoPlugin', copy);
+    compiler.hooks.watchRun.tap('CopyMonacoPlugin', copy);
+  }
+}
```

Comment:
need to make sure we do this the "right" way we want. Can't use the `afterEnvironment` hook that the `CopyUniconsPlugin` uses because the files are copied to the build dir which is deleted after that hook...